field_select.go

```diff
@@ -514,16 +514,22 @@ func (s *Select[T]) updateViewportHeight() {
 }
 
 func (s *Select[T]) activeStyles() *FieldStyles {
-	theme := s.theme
-	if theme == nil {
-		theme = ThemeCharm()
-	}
+	theme := s.Theme()
 	if s.focused {
 		return &theme.Focused
 	}
 	return &theme.Blurred
 }
 
+// Theme returns the theme of the field.
+func (s *Select[T]) Theme() *Theme {
+	theme := s.theme
+	if theme == nil {
+		theme = ThemeCharm()
+	}
+	return theme
+}
```

Comment:
```suggestion
	if s.theme != nil {
		return s.theme
	}
	return ThemeCharm()
}
```

feels a bit better I think 🤔